user/configure-device.rst

```diff
@@ -17,43 +17,11 @@ Refer to the  `OpenWISP 2 installation tutorial
 2. Install OpenWRT on VirtualBox
 --------------------------------
 
-.. note::
-    **You can skip this part if you plan on installing openwisp-config on
-    a physical device.**
-
-Download ``combined-ext4.img.gz`` from `this
-page <https://downloads.openwrt.org/releases/18.06.1/targets/x86/64/>`__.
-The other images may not work well.
-
-Extract the downloaded file and convert the image file to a `VirtualBox
-<https://www.virtualbox.org/>`_ disk:
-
-.. code-block:: bash
-
-    VBoxManage convertfromraw --format VDI \
-    openwrt-18.06.1-x86-64-combined-ext4.img \
-    openwrt-18.06.1-x86-64-combined-ext4.vdi
-
-If you encounter an error about ``VERR_ID_INVALID_SIZE``, you need to
-pad the image with the following command:
-
-.. code-block:: bash
-
-    dd if=openwrt-18.06.1-x86-64-combined-ext4.img \
-    of=openwrt-18.06.1-x86-64-padded.img bs=128000 conv=sync
-
-And then try to convert the padded image using the previous command.
-
-Next, open up VirtualBox and create a VM. Load up the VDI and then start
-the machine. If done correctly, it should boot to a GRUB menu and then
-proceed with initialization. When the text stops scrolling, simply press
-Enter to activate the terminal.
+Follow the `OpenWrt on VirtualBox Howto
+<https://openwrt.org/docs/guide-user/virtualization/virtualbox-vm>`_.
```

Comment:
were you able to get it working by following that link?
I remember in the past users had problems.

Reply:
Yes I did! It surprised me that the instructions were similar. Maybe the problems were fixed.